src/main/java/appeng/integration/modules/ImmibisMicroblocks.java

```diff
@@ -30,7 +30,8 @@
 import net.minecraft.world.World;
 
 import mods.immibis.core.api.multipart.ICoverSystem;
-import mods.immibis.core.api.multipart.IMultipartTile;
+import mods.immibis.core.api.multipart.ICoverableTile;
+import mods.immibis.core.api.multipart.IPartContainer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'mods.immibis.core.api.multipart.IPartContainer'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)